src/images/yldsvg_dark.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you need this line?

Reply:
No. I'm not even using this component. I may remove the whole thing.